impl/src/templates/crate.BUILD.template

```diff
@@ -39,7 +40,5 @@ load(
 {%- include "templates/partials/rust_library.template" %}
 {%- elif target.kind == "rlib" %}
 {%- include "templates/partials/rust_library.template" %}
-{%- else %}
-# Unsupported target "{{ target.name }}" with type "{{ target.kind }}" omitted
```

Comment:
What's the rationale for removing this line? I added it here because I thought it might be unclear to people why some targets are missing. This note gives them something to search for.

Reply:
Ah sorry for not including that. Not sure if I did anything wrong but this line seemed to comment out the start of a rule (`rust_binary`) in the `cc` crate (iirc). The templates were with LF line endings and adding a new line after the unsupported target line did not work.

Reply:
Can you provide me an example of a generated BUILD file that does this?

the "{%-" has some semantic meaning around keeping or removing newlines, and might be related.  Unclear to me why only this target would cause you issues though since the other lines use the same thing. Your BUILD file should help.

Reply:
The BUILD file causing the issue is here: https://gist.github.com/mstg/6991d07372dabf4aeb64a5c664b879a4

Reply:
If this only occurred after adding L32 (the set), remove "-" from the initial "{%". You might be removing a required newline inside of this for loop. Seems likely that the spacing among these partials is inconsistent enough that you might be fine removing that extra newline in most cases, but not the "[.., unsupported, rust_binary, ...]" case.

Reply:
I fixed this by adding a new empty line on top of `rust_binary.template`. I'll add those changes together with merged #93.